src/backend/joanie/tests/core/api/admin/orders/test_refund.py

```diff
@@ -327,7 +328,8 @@ def test_api_admin_orders_refund_an_order(self):
             f"Withdrawn on {order.payment_schedule[3]['due_date'].strftime('%m/%d/%Y')}",
             "Canceled",
             "Total €100.00",
-            f"This mail has been sent to {order.owner.email} by Test Catalog [None]",
+            f"This mail has been sent to {order.owner.email} by Test Catalog \
+                [https://richie.education]",
```

Comment:
Please use the same existing style.
```suggestion
            f"This mail has been sent to {order.owner.email}",
            "by Test Catalog [https://richie.education]",
```